decisions/0002-new-instructor-org-association.rst

```diff
@@ -0,0 +1,32 @@
+1. New Instructors Organization Association
+----------------------------------------------
+
+Status
+------
+
+Initial
```

Comment:
Initial?

Reply:
Yeah, I followed the other ADR as a template